src/mcts/search.cc

```diff
@@ -260,7 +280,8 @@ std::vector<std::string> Search::GetVerboseStats(Node* node,
     }
     oss << ") ";
 
-    if (edge.IsTerminal()) oss << "(T) ";
+    oss << " C:" << std::bitset<8>(edge.edge()->GetCertaintyState());
```

Comment:
Instead of pritting bitset, it would be nice to build some string like `..W..T.D` (with names of flags set).
Not necessary, just easier to interpret.